assets/src/edit-story/components/library/panes/text/textSets/textSetsPane.js

```diff
@@ -92,9 +92,23 @@ function TextSetsPane({ paneRef }) {
 
   const addIdToTextSets = useCallback(
     (textSetsArray = []) =>
-      textSetsArray.map((elements) => {
+      textSetsArray.map(({ textSetFonts = [], textSetCategory, elements }) => {
         return {
           id: `text_set_${uuidv4()}`,
+          title: sprintf(
+            /* translators: 1: text set category. 2: list of fonts. */
+            _n(
+              'Text set %1$s with %2$s font',
+              'Text set %1$s with %2$s fonts',
+              textSetFonts.length,
+              'web-stories'
+            ),
+            textSetCategory,
```

Comment:
What are some example values for `textSetCategory`?

Reply:
![Screen Shot 2021-04-14 at 11 08 42 AM](https://user-images.githubusercontent.com/10720454/114758429-cf214100-9d11-11eb-88f2-6d87f1b6b15d.png)

Reply:
oooo let me map those to the constants so they translate